subsys/bluetooth/controller/ll_sw/ull_conn_types.h

```diff
@@ -505,6 +504,18 @@ struct ll_conn {
 	/* Detect empty L2CAP start frame */
 	uint8_t  start_empty:1;
 #endif /* CONFIG_BT_CTLR_LLID_DATA_START_EMPTY */
+
+	/*
+	** EGON TODO these were added to the upstream ll_conn structure
+	** leaving them here for now as a reminder that this may cause
+	** issues for us
+	struct node_tx *tx_head;
+	struct node_tx *tx_ctrl;
+	struct node_tx *tx_ctrl_last;
+	struct node_tx *tx_data;
+	struct node_tx *tx_data_last;
+	*/
+
```

Comment:
Somethings not right, these have long ago been replaced with the ull_tx_q object in line 439, The legacy ones are at line 311.

Reply:
Agreed that they are replaced with ull_tx_q. I guess we don't need to keep them as a reminder here anymore. I'll remove them